core/field_dropdown.js

```diff
@@ -140,6 +140,13 @@ Blockly.FieldDropdown.prototype.imageElement_ = null;
  */
 Blockly.FieldDropdown.prototype.imageJson_ = null;
 
+/**
+ * A reference to the currently selected menu item.
+ * @type {Blockly.MenuItem}
+ * @private
+ */
+this.selectedMenuItem_ = null;
```

Comment:
You'll need to set this at the bottom of the constructor. It won't work like that.

Reply:
Done!